app/components/Comments/CommentView.js

```diff
@@ -26,28 +31,38 @@ const CommentView = (props: Props) => {
     user,
     loggedIn,
     style,
-    displayTitle = true
+    displayTitle = true,
+    newOnTop = false
   } = props;
   const commentFormProps = { commentTarget, user, loggedIn };
   const tree = generateTreeStructure(comments);
 
   return (
     <div style={style}>
-      {displayTitle && <h3>Kommentarer</h3>}
-      <LoadingIndicator loading={!comments}>
-        {comments && (
-          <CommentTree comments={tree} commentFormProps={commentFormProps} />
-        )}
-      </LoadingIndicator>
+      <Title displayTitle={displayTitle} />
+      <Flex
+        style={{
+          flexDirection: newOnTop ? 'column-reverse' : 'column'
+        }}
+      >
+        <LoadingIndicator loading={!comments}>
+          {comments && (
+            <CommentTree
+              comments={newOnTop ? tree.reverse() : tree}
```

Comment:
Yeah @edvardvb. It reverses the first layer of comments in the generated tree, not the rest of them.